### PR TITLE
Fixes #19827: Remove CFEngine enterprise specific policies

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/19827

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: techniques/system/rudder-service-relayd/1.0/relay/propagatePromises.cf [hunk 1]
```diff
@@ -88,8 +88,7 @@ bundle agent propagate_promises(trackingkey, technique_name, relay_sync_method,
                                                   "could_not_propagate_masterfiles",
                                                   "could_not_propagate_ncf_local_promise",
                                                   "could_not_propagate_ncf_common_promise",
-                                                  "nodeslist_error",
-                                                  "could_not_copy_license"
```
Comment (peckpeck) [inline techniques/system/rudder-service-relayd/1.0/relay/propagatePromises.cf:92]: I don't see the removal of this bundle

File: techniques/system/rudder-service-relayd/1.0/relay/propagatePromises.cf [hunk 2]
```diff
@@ -151,13 +149,6 @@ bundle agent propagate_promises(trackingkey, technique_name, relay_sync_method,
         comment      => "Copying list of nodes under this policy server",
         classes      => classes_generic("rudder_relayd_nodeslist");
 
-    enterprise_edition::
-      "${sys.workdir}/masterfiles/license.dat"
-        copy_from    => local_cp("${sys.workdir}/inputs/license.dat"),
-        comment      => "Putting the right license in the bootstrap",
-        classes      => if_else("license_copied", "could_not_copy_license");
```
Comment (amousset) [inline techniques/system/rudder-service-relayd/1.0/relay/propagatePromises.cf:158]: here

File: techniques/system/rudder-service-relayd/1.0/relay/propagatePromises.cf (hunk 1 above)
Comment (amousset) [reply techniques/system/rudder-service-relayd/1.0/relay/propagatePromises.cf:92]: https://github.com/Normation/rudder-techniques/pull/1719/files#r692873535

**Review by peckpeck — APPROVED** [review]

**Pull request merged**